package.json

```diff
@@ -36,6 +36,8 @@
 		"typedoc": "typedoc src/index.ts",
 		"api:check": "npm run compile && api-extractor run",
 		"api:update": "npm run compile && api-extractor run --local",
+		"setup-hooks": "chmod +x scripts/install-git-hooks.sh && ./scripts/install-git-hooks.sh",
+		"uninstall-hooks": "git config --local --unset core.hooksPath && rm -rf .githooks",
```

Comment:
Even thought the `rm` is restricted to the `.githooks` folder, a scripted `rm -rf` makes me pretty nervous... and could accidently erase other hook actions in the future. Maybe remove should be manual only?